RevenueCatUI/CustomerCenter/Views/FeedbackSurveyView.swift

```diff
@@ -83,33 +81,20 @@ struct FeedbackSurveyButtonsView: View {
     var loadingState: String?
 
     var body: some View {
-        VStack(spacing: Self.buttonSpacing) {
-            ForEach(options, id: \.id) { option in
-                AsyncButton(action: {
-                    await self.onOptionSelected(option)
-                }, label: {
-                    if self.loadingState == option.id {
-                        TintedProgressView()
-                    } else {
-                        Text(option.title)
-                    }
-                })
-                .buttonStyle(ProminentButtonStyle())
-                .disabled(self.loadingState != nil)
-            }
+        ForEach(options, id: \.id) { option in
+            AsyncButton(action: {
+                await self.onOptionSelected(option)
+            }, label: {
+                if self.loadingState == option.id {
+                    TintedProgressView()
+                } else {
+                    Text(option.title)
+                }
+            })
```

Comment:
I know this was there before but I got this same suggestion in one of my PRs

```suggestion
            AsyncButton {
                await self.onOptionSelected(option)
            } label: {
                if self.loadingState == option.id {
                    TintedProgressView()
                } else {
                    Text(option.title)
                }
            }
```